docs/concepts/what-is-a-temporal-cluster.md

```diff
@@ -32,3 +32,40 @@ The database stores the following types of data:
   For production environments, we recommend using Elasticsearch.
 
 An Elasticsearch database can be added to enable [Advanced Visibility](/concepts/what-is-advanced-visibility).
+
+**Versions**
+
+Temporal tests compatibility by spanning the **minimum** and **maximum** stable non-EOL major versions for each supported database.
+As of time of writing, these specific versions used in our test pipelines and actively tested before we release any version of Temporal:
+
+- **Cassandra v3.11 and v4.0**
+- **PostgreSQL v10.18 and v13.4**
+- **MySQL v5.7 and v8.0** (specifically 8.0.19+ due to a bug)
+
+We will update these support ranges once a year and the release notes of each Temporal Server will declare when we plan to drop support for database versions reaching End of Life.
+Since Temporal Server primarily relies on core database functionality, we do not expect compatibility to break often.
+
+- We only rely on core database features, so compatibility should not break often.
```

Comment:
```suggestion
We update these support ranges once a year.
The release notes of each Temporal Server declare when we plan to drop support for database versions reaching End of Life.

- Because Temporal Server primarily relies on core database functionality, we do not expect compatibility to break often.
```